scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ix_aern_mnk.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Area: Grand Palace of HuXzoi
+-- NPC:  Ix_aern_mnk
+-- ID: 16916815
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    local chance = GetServerVariable("[SEA]IxAern_DropRate"); -- Adjust drop rate for the items based on the organs traded.

```

Comment:
A server vars purpose is to save data between restarts. If that's not what
you're doing (it's not) then I will fix local vars for what you need.